quickstart-mongodbatlas/README.md

```diff
@@ -0,0 +1,29 @@
+quickstart-mongodbatlas
+=============
+## MongoDB Atlas on the AWS Cloud
+
+Start here: [beta-launch.md](beta-launch.md)
+
+This Quick Start sets up a flexible, scalable AWS environment for MongoDB Atlas through AWS CloudFormation resources.
+
+MongoDB Atlas is an open source, NoSQL database that provides support for JSON-styled, document-oriented storage systems. 
+Its flexible data model enables you to store data of any structure, and it provides full index support, sharding, and replication.
```

Comment:
Think I commented on this before but NoSQL is not applicable with MongoDB being general purpose, also the description is more about the database vs Atlas.  I recommend w/ this stuff to use pre-existing content or new contact created with Product Marketing, for example: "MongoDB Atlas is the global cloud database service for modern applications. Deploy fully managed MongoDB across AWS, Azure, or GCP. Best-in-class automation and proven practices guarantee availability, scalability, and compliance with the most demanding data security and privacy standards."

Reply:
sure - we can put whatever, again this is cut and paste from the legacy quickstart.